---
layout: cv
title: CV | Unai Alaña
---
# Unai Alaña Arranz
Business Data Analytics & Data Science

<div id="webaddress">
<a href="mailto:unaialanaa@gmail.com">unaialanaa@gmail.com</a>
|
<i class="fa fa-github"></i> <a href="http://github.com/ualana">ualana</a>
|
<i class="fa fa-linkedin"></i> <a href="https://es.linkedin.com/in/unai-ala%C3%B1a">Unai Alaña</a>
</div>


## Sobre mi

Actualmente trabajo como **Técnico-Programador Analytics** en LABORAL Kutxa, una entidad financiera de referencia en el País Vasco. Me apasiona el mundo de los datos y entender cómo exprimirlos en distintos contextos para generar valor y tomar decisiones informadas. En mi rato libre me gusta andar en bici y tomar algo con mi grupo de amigos.

## Empleo

`2024-act.` 
__LABORAL Kutxa__ Senior Bioinformatics Scientist (software engineer and solutions architect). 
> My work includes integrating Illumina's best-in-class _DRAGEN_ bioinformatics analysis pipelines with our flagship cloud platforms (_BaseSpace Sequence Hub_ and _Illumina Connected Analytics_). I also develop public tools such as the BaseSpace Sequence Hub Commandline Interface, alongside many other internal software engineering projects ranging from bioinformatics scripts to web applications.

`2023-2024` 
__LABORAL Kutxa__ Becario (Departamento de Analytics).
> I joined Illumina's R&D organisation in 2015 where I worked on a wide range of bioinformatics and data science projects. Some public-facing work focused on variant comparison and analysis tooling, and I also advanced the popular _Platinum Genomes_ gold-standard variant truthset by introducing novel validation methods.

## Skills

* Python
* SQL
* R
* HTML & CSS
* Power BI
* Tableau
* Looker (Google Data Studio)
* Git
* Salesforce.com
* Salesforce Marketing Cloud
* Linux
* AWS (EC2)


## Educación

`2023`
__University of Essex__ Data Science (_International Student Program_)

`2020-2024`
__Universidad de Mondragon__ Business Data Analytics

### Premios

`2022` Datathon Open Data Bizkaia - Transformar Bizkaia a través de sus datos (_2º posición_)

## Frameworks

* Pandas
* Tensorflow
* Matplotlib
* scikit-learn
* SFMC
* SSIS
* Teradata
* SQL Server
* Office 365

## Proyectos

### Universidad
`2019`
Krusche P, Trigg L, Boutros PC, Mason CE, De La Vega FM, __Moore BL__ _et al._ Best practices for benchmarking germline small-variant calls in human genomes. _Nature Biotechnology_, 37:555–56.

`2017`
Peluso S, Douglas A, Hill A, De Angelis C, __Moore BL__ _et al._ Fibroblast growth factors (FGFs) prime the limb specific Shh enhancer for chromatin changes that balance histone acetylation mediated by E26 transformation-specific (ETS) factors. _eLife_, 6:e28590.

Eberle MA, Fritzilas E, Krusche P, Källberg M, __Moore BL__ _et al._ A reference data set of 5.4 million phased human variants validated by genetic inheritance from sequencing a three-generation 17-member pedigree. _Genome Research_, 27:157-164.

`2016`
Yu N, Nützmann H-W, MacDonald JT, __Moore B__ _et al._ Delineation of metabolic gene clusters in plant genomes by chromatin signatures. _Nucleic Acids Research_, 44(5):2255-2265.

Welch L, Brooksbank C, Schwartz R, Morgan SL, Gaeta B, Kilpatrick AM, Meitchen D, __Moore BL__ _et al._ Applying, Evaluating and Refining Bioinformatics Core Competencies. _PLoS Compututational Biology_, 12(5): e1004943.

`2015`
__Moore BL__ _et al._ Integrative modeling reveals the principles of multi-scale chromatin boundary formation in human nuclear organization. _Genome Biology_, 16:110.

Fraser J, Ferrai C, Chiariello AM, Schueler M, Rito T, Laudanno G, Barbieri M, __Moore BL__ _et al._ Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation. _Molecular systems biology_, 11:852.

`2013`
__Moore BL__ _et al._ High–quality protein backbone reconstruction from alpha carbons using Gaussian mixture models. _Journal of computational chemisty_, 34(22):1881-9.

`2011`
Gardner PP, Daub J, Tate J, __Moore BL__ _et al._ Rfam: Wikipedia, clans and the “decimal” release. _Nucleic acids research_, 39(S1):D141-5.

### Personal

`2017`
__Moore BL__, Semple CA. Crossing Borders: Modeling Approaches to Understand Chromatin Domains and Their Boundaries. _Epigenetics and Systems Biology_, 3:45-67.

## Non-academic

`2014` Pieces in _The Huffington Post_ (US): Here Are the Most Overrated and Underrated Movies of All Time; Celebrity Twitter Followers, by Gender ([huffingtonpost.com/benjamin-moore](http://www.huffingtonpost.com/benjamin-moore/))

`2006–` Articles for _English Wikipedia_: European Nucleotide Archive, RNA thermometer, Toxin-antitoxin system and more ([en.wikipedia.org/wiki/User:Ben_Moore](https://en.wikipedia.org/wiki/User:Ben_Moore)).

## Projects

### github

My [personal github account](https://github.com/blmoore) hosts my hobby development projects as well as listing contributions to open source tools. Examples include: shiny apps [blackspot](https://github.com/blmoore/blackspot) and [shinybrot](https://github.com/blmoore/shinybrot), as well as an R package hosted by CRAN ([mandelbrot](https://github.com/blmoore/mandelbrot)).

### blog

I write a data science blog at [blm.io](http://blm.io/blog) where I apply R and Python to open datasets and write-up the results. My posts have been featured by sites including FiveThirtyEight, BuzzFeed, AVclub, io9 and more, and also led to being an invited blogger at the Huffington Post. My original wordpress blog ([benjaminlmoore.wordpress.com](http://benjaminlmoore.wordpress.com)) has received over 150,000 unique visitors from 180 countries.

### EdinbR

I co-founded a usergroup for the R programming language in Edinburgh called EdinbR. I helped to organise our monthly meetings and built our website: [edinbr.org](http://edinbr.org). Our meetings attract 30-50 statisticians, data scientists and developers for talks and discussion about the R language and its applications. We're sponsored by Revolution Analytics and have an organisational github account at [github.com/EdinbR](https://github.com/EdinbR).

### datarea

I entered Imperial College's [Summer Data Challenge](https://www.imperial.ac.uk/data-science/education/summer-data-challenge/) competition, where entrants analysed a given dataset and then proposed a startup idea based on their results. I developed a modelling technique to select housing areas for investment and was awarded third place: £2,000 and startup support from Imperial Create Lab. My entry can be seen at [blm.io/datarea](http://blm.io/datarea) and the code is shared on my github account.

## Placements

`2010-11`
__Wellcome Trust Sanger Institute__ (_Cambridge, UK_)
Worked for two summers on the Rfam database ([rfam.xfam.org](http://rfam.xfam.org)) in the Bateman group


## Links

<!-- fa are fontawesome, ai are academicons -->
* <i class="fa fa-envelope"></i> <a href="mailto:unaialanaa@gmail.com">unaialanaa@gmail.com</a><br />
* <i class="fa fa-github"></i> <a href="http://github.com/ualana">ualana</a><br />
<!--* <i class="fa fa-twitter"></i> <a href="http://twitter.com/twitter">Twittter</a><br /> -->
* <i class="fa fa-linkedin"></i> <a href="https://es.linkedin.com/in/unai-ala%C3%B1a)/">LinkedIn</a>
<!-- * <i class="fa fa-stack-overflow"></i> <a href="http://stackoverflow.com/users/1274516/blmoore">StackOverflow</a> -->
<!-- * <i class="fa fa-wikipedia"></i> <a href="https://en.wikipedia.org/wiki/User:Ben_Moore">Wikipedia Userpage</a><br /> -->
<!-- * <i class="ai ai-google-scholar"></i> <a href="http://scholar.google.com/citations?user=YMxsGpsAAAAJ">Google scholar</a> -->
<!-- * <i class="ai ai-orcid"></i> <a href="https://orcid.org/0000-0002-4074-1933">ORCiD</a> -->
<!-- * <i class="ai ai-figshare"></i> <a href="https://figshare.com/authors/Benjamin_Moore/99461">figshare</a> -->

<!-- ### Footer

Last updated: Jan 2025 -->
